src/lib/device-logs/backends/redis.ts

```diff
@@ -134,8 +140,8 @@ export class RedisBackend implements DeviceLogsBackend {
 		return this.cmds.connected && this.pubSub.connected;
 	}
 
-	private getKey(ctx: LogContext) {
-		return `device:${ctx.id}:logs`;
+	private getKey(ctx: LogContext, suffix?: string) {
+		return `device:${ctx.id}:${suffix ?? 'logs'}`;
```

Comment:
Can you change to this before merging please as it'll improve automated docs when type hinting and is clearer that the param is always defaulted
```suggestion
	private getKey(ctx: LogContext, suffix = 'logs') {
		return `device:${ctx.id}:${suffix}`;
```

Reply:
sure thing, pushing now